src/Chainweb/Backup.hs

```diff
@@ -9,110 +9,109 @@
 {-# LANGUAGE TypeOperators #-}
 
 module Chainweb.Backup
-    ( BackupEnv(..)
-    , BackupOptions(..)
-    , BackupStatus(..)
-    , makeBackup
-    , checkBackup
-    ) where
-
-import Control.Lens
+  ( BackupEnv (..),
+    BackupOptions (..),
+    BackupStatus (..),
+    makeBackup,
+    checkBackup,
+  )
+where
 
+import Chainweb.ChainId
+import Chainweb.Logger
+import Chainweb.Pact.Backend.Types
+import Chainweb.Pact.Backend.Utils (chainDbFileName, withSqliteDb)
+import Chainweb.Storage.Table.RocksDB
+import Chainweb.Utils
 import Control.Concurrent.Async
+import Control.Lens
 import Control.Monad
 import Control.Monad.Catch
-import Data.HashSet(HashSet)
+import Data.HashSet (HashSet)
 import Data.String
 import qualified Data.Text as T
 import qualified Data.Text.IO as T
 import qualified Data.Text.Lazy as TL
 import qualified Data.Text.Lazy.Encoding as TL
+import Pact.Types.SQLite
+import Servant
 import System.Directory
 import System.FilePath
 import System.LogLevel
 
-import Pact.Types.SQLite
-import Servant
-
-import Chainweb.ChainId
-import Chainweb.Logger
-import Chainweb.Pact.Backend.Types
-import Chainweb.Pact.Backend.Utils(chainDbFileName, withSqliteDb)
-import Chainweb.Utils
-
-import Chainweb.Storage.Table.RocksDB
-
 data BackupOptions = BackupOptions
-    { _backupIdentifier :: !FilePath
-    , _backupPact :: !Bool
-    }
+  { _backupIdentifier :: !FilePath,
+    _backupPact :: !Bool
+  }
 
 data BackupEnv logger = BackupEnv
-  { _backupRocksDb :: !RocksDb
-  , _backupDir :: !FilePath
-  , _backupPactDbDir :: !FilePath
-  , _backupChainIds :: !(HashSet ChainId)
-  , _backupLogger :: !logger
+  { _backupRocksDb :: !RocksDb,
+    _backupDir :: !FilePath,
+    _backupPactDbDir :: !FilePath,
+    _backupChainIds :: !(HashSet ChainId),
+    _backupLogger :: !logger
```

Comment:
Ooof. Does ormolu hate leading commas?

Reply:
Seems so. Doesn't appear to be too configurable.